content/applications/hr/appraisals/goals.rst

```diff
@@ -0,0 +1,73 @@
+=====
+Goals
+=====
+
+The ability to set goals for employees to work toward is an important feature of Odoo *Appraisals*.
+Goals are typically set during an appraisal, so the employee knows what to work toward before their
+next appraisal.
+
+To view all goals, navigate to :menuselection:`Appraisals application --> Goals`. This presents all
+the goals for every employee, in a default Kanban view.
+
+Each goal card contains the following information:
+
+- :guilabel:`Goal`: the name of the goal.
+- :guilabel:`Employee`: the employee the goal is assigned to.
+- :guilabel:`Activities`: displays the corresponding :doc:`activity icon
+  <../../essentials/activities>` for the record.
+- :guilabel:`Deadline`: the due date for the goal.
+- :guilabel:`Progress`: the percentage of competency set for the goal.
+- :guilabel:`Employee Icon`: the profile icon of the employee the goal is assigned to.
+
+If a goal is completed, a :guilabel:`Done` banner appears in the top-right corner of the goal card.
+
+.. image:: goals/goals.png
+   :align: center
+   :alt: The goals Kanban view, with nine goal cards.
+
+.. note::
+   Every individual goal requires its own record for each employee. If multiple employees have the
+   same goal, a goal card for each employee appears on the list.
+
+   For example, if both Bob and Sara have the same goal of `Typing`, two cards appear in the Kanban
+   view: one `Typing` goal for Bob, and another `Typing` goal for Sara.
+
+New goal
+========
+
+To create a new goal, navigate to :menuselection:`Appraisals application --> Goals`, and click
+:guilabel:`New` top-left corner to open a blank goal form.
+
+Enter the information on the goal form. The information requested is all the same information that
+appears on the goal card  in the Kanban view, with the addition of a :guilabel:`Tags` field and a
```

Comment:
> Enter the information on the goal form.

Can be a little bit more specific. How about something like:

"Input the Goal, Employee, Progress, and Deadline information as discussed in the [goal card section] (ref link to the earlier part" ?